setup.py

```diff
@@ -51,22 +51,32 @@ def get_install_requires():
 def get_extras_require():
     # type: () -> Dict[str, List[str]]
 
+    _major, _sub = sys.version_info[:2]
+    _fastai_available = (_major == 3) and (_sub > 5)
+
+    testing_requirements = [
+        'bokeh', 'chainer>=5.0.0', 'cma', 'keras', 'lightgbm', 'mock',
+        'mpi4py', 'mxnet', 'pandas', 'plotly>=4.0.0', 'pytest', 'scikit-optimize',
+        'tensorflow', 'tensorflow-datasets', 'xgboost', 'scikit-learn>=0.19.0',
+        'torch', 'torchvision', 'pytorch-ignite', 'pytorch_lightning',
+    ]
+
+    example_requirements = [
+        'chainer', 'keras', 'catboost', 'lightgbm', 'scikit-learn',
+        'mxnet', 'xgboost', 'torch', 'torchvision', 'pytorch-ignite',
+        'pytorch-lightning', 'dask-ml', 'dask[dataframe]',
+        # TODO(Yanase): Update examples to support TensorFlow 2.0.
+        # See https://github.com/pfnet/optuna/issues/565 for further details.
+        'tensorflow<2.0.0',
+    ]
+    if _fastai_available:
+        testing_requirements.append("fastai==1.0.58")
```

Comment:
This version specification may be too much.

Reply:
I think so, too. > may be too much
How about replacing the version specification with "fastai>=1.0.0"?

Reply:
As I don't know when the v2 is available, I instead would like to limit fastai<2

Ref: https://github.com/pfnet/optuna/pull/585#discussion_r338861184

Reply:
> I instead would like to limit fastai<2

Sounds good.